crates/re_renderer/src/config.rs

```diff
@@ -23,6 +26,17 @@ pub enum DeviceTier {
     //HighEnd
 }
 
+/// Type of Wgpu backend.
+///
+/// Used in the rare cases where it's necessary to be aware of the api differences between
+/// wgpu-core and webgpu.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum WgpuBackendType {
+    WgpuCore,
+    #[cfg(web)]
+    WebGpu,
```

Comment:
Where does WebGL fit in?

Reply:
into `WgpuCore` as it is a backend that is implemented in wgpu_core. will make a comment